### PR TITLE
kyc request status is not "completing" after finishing the aplyid verification

### PR DESCRIPTION
**Pull request by eoliveros**
- [x] added csrf.exempt on the kyc_endpoint blueprint

### CODE REVIEW

File: src/kyc_endpoint.py [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
-from app_core import app, db, limiter
+from app_core import app, db, limiter, csrf
 from models import KycRequest
 import websocket
 import kyc_core
 
 logger = logging.getLogger(__name__)
 kyc = Blueprint('kyc', __name__, template_folder='templates')
 limiter.limit("100/minute")(kyc)
+csrf.exempt(kyc)
```
Comment (djpnewton) [inline src/kyc_endpoint.py:13]: I think we just want to exempt the one `/aplyid_webhook` route

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: src/kyc_endpoint.py (hunk 1 above)
Comment (eoliveros) [reply src/kyc_endpoint.py]: apply the change.

File: src/kyc_endpoint.py [hunk 2]
```diff
@@ -60,6 +60,7 @@ def aplyid_pdf(token=None):
     response.headers.set('Content-Type', 'application/pdf')
     return response
 
+@csrf.exempt
```
Comment (djpnewton) [inline src/kyc_endpoint.py:63]: documentation seems to show the exemption *after* the route (https://flask-wtf.readthedocs.io/en/0.15.x/csrf/#exclude-views-from-protection)

**Review by djpnewton — CHANGES_REQUESTED** [review]

**Review by djpnewton — COMMENTED** [review]
I think this looks good. Have you tested the changes?

**Comment by eoliveros** [comment]
> I think this looks good. Have you tested the changes?

the only way to test this is on the production server. i made a manual change yesterday to get this to work and made the changed it back, since its a manual change.


retesting it again:
image

before:
image

after:
image

seems to work

**Review by djpnewton — APPROVED** [review]

**Pull request merged**